staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -86,15 +94,17 @@ func Register() {
 	registerMetrics.Do(func() {
 		legacyregistry.MustRegister(etcdRequestLatency)
 		legacyregistry.MustRegister(objectCounts)
-		legacyregistry.MustRegister(dbTotalSize)
+    legacyregistry.MustRegister(etcdObjectCounts)
+    etcdObjectCountstry.MustRegister(dbTotalSize)
```

Comment:
you may need to run gofmt and golint.

IIRC: 
```shell
./hack/update_gofmt.sh
./hack/update_golint.sh
```

Reply:
Done